PR-CHECKLIST

```diff
@@ -0,0 +1,19 @@
+# Checklist for merging PRs
```

Comment:
Since this is using markdown syntax perhaps this should be renamed to `PR-CHECKLIST.md`? Perhaps it belongs in the `docs/` directory as well?

Or is this some kind of 'magic' file that GitHub picks up and presents in the UI.

Reply:
Renamed, that was a mistake ;)